### ISSUE TITLE
How to model χάρις ὑμῖν καὶ εἰρήνη

### ISSUE THREAD

**Issue by jtauber** [biblicalhumanities/treedown#19]
This type of construction is interesting to model because of the apparent gapping.

In English, this is often translated "grace and peace to you" because, in English, that's taken as "[grace and peace] to you" and less likely "grace and [peace to you]".

But it seems in Greek, χάρις καὶ εἰρήνη ὑμῖν WOULD be read as χάρις καὶ [εἰρήνη ὑμῖν] so in order to convey [χάρις καὶ εἰρήνη] ὑμῖν you have to say χάρις ὑμῖν καὶ εἰρήνη.

In other words, it would be nice to convey that ὑμῖν is modifying not just χάρις but εἰρήνη. Or if you don't think it does (i.e. the translation "grace and peace to you" is wrong and it should actually be translated "grace to you and peace") then it would be nice to convey that interpretation unambiguously too.

**Comment by mwpalmer** [comment]
Agreed.

**Comment by jtauber** [comment]
Actually this is interesting for other reasons too, as pointed out to me by Giuseppe Celano.

It's not just the conjunction (although Giuseppe gave another example in English: "all men and women"  and the challenge of properly conveying the scope of the quantifier as "all [men and women]" not "[all men] and women")

But χάρις ὑμῖν καὶ εἰρήνη is doubly interesting because, as Giuseppe pointed out to me, there's an implied verb. Without the implied verb (related to a passive verb of giving), you can't explain why ὑμῖν is dative. 

The English expression "Happy Birthday to you" is another example of this (so it would be interesting to see how that is normally analyzed). Also things like "Best wishes to you and your family".

So there are TWO things going on. The scoping of the ὑμῖν in light of the conjunction and the implied verb of giving.
